src/bitcoin-cli.cpp

```diff
@@ -528,6 +536,10 @@ class NetinfoRequestHandler : public BaseRequestHandler
                     strprintf("%s%s", peer.is_bip152_hb_to ? "." : " ", peer.is_bip152_hb_from ? "*" : " "),
                     m_max_age_length, // variable spacing
                     peer.age,
+                    m_max_addr_rate_limited_length, // variable spacing
+                    peer.addr_rate_limited ? ToString(peer.addr_rate_limited) : "",
```

Comment:
Wouldn't it be better to print `0` if the value is zero, instead of an empty string?

```suggestion
                    peer.addr_rate_limited,
```

Reply:
Yes, that was done to optimise signal to noise.  Examples follow; keep in mind that they are of much smaller nodes than typical longer-running ones that can have 125-250 peer connections. 

![Screenshot from 2021-07-27 19-24-53](https://user-images.githubusercontent.com/2415484/127214893-6228644e-4f4b-4f8b-9173-50e9dd511996.png)

![Screenshot from 2021-07-27 19-24-08](https://user-images.githubusercontent.com/2415484/127214885-68d4fe38-1c23-45b8-a8b8-87e770a2d5cd.png)